synapse/cortex.py

```diff
@@ -1756,7 +1756,7 @@ def filtercmpr(sode):
             tagprops = sode.get('tagprops')
             if tagprops is None:
                 return False
-            return tagprops.get((tag, prop)) is not None
+            return tagprops.get(tag, {}).get(prop) is not None
```

Comment:
Plz make these not do the default dict values get constructed by default ( here and others below ) :D